src/widgets/button.rs

```diff
@@ -128,12 +132,12 @@ pub fn button_render(
                             bottom: Units::Stretch(1.0).into(),
                             left: Units::Stretch(1.0).into(),
                             right: Units::Stretch(1.0).into(),
+                            font_size: StyleProp::Value(font_size),
                             ..Default::default()
                         }}
                         text={TextProps {
```

Comment:
I think `TextProps` shouldn't accept a `font_size` as it should auto select the font size from the style props which casscade down the tree see:
https://github.com/StarArawn/kayak_ui/blob/main/src/styles/style.rs#L423

I don't think we should make that change in this PR though. 👍